rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -27,6 +28,46 @@ The *Recording Toolbar* appears on the screen when RPA Recorder starts recording
 
 When a recording state changes, a toast notification appears indicating current state. The notification stays for two seconds and disappears automatically.
 
+== Image-Based Mouse Clicks
+
+Use image-based mouse clicks at any point during a normal recording session to record mouse clicks on a UI element that otherwise cannot be recorded properly. This feature guides you through the mouse click process. An image on the screen and mouse clicks must be captured, and the mouse clicks are recorded relative to the captured image.
+
+The feature also checks your display settings automatically while recording to ensure the feature records correctly.
+
+This feature will result in _Search Pattern_ and _Mouse Action_ Action Steps when the workflow is generated in RPA Builder from the recording.
+
+When using this feature:
+
+* The Recording Toolbar is inactive.
+* Keyboard events are not recorded.
+
+=== Recommended Display Settings
+
+Apply the following recommended display settings to capture image-based mouse clicks:
+
+* Single monitor. 
++
+Although the image-based mouse clicks feature supports multiple monitors, that setting limits the capturing to the primary display for Bot tasks and the web page for Web tasks.
+* Display scaling of 100%. If the scaling is different, then there is a risk that the captured image will not be found by the workflow generated in RPA Builder from the recording.
+
+To check your display settings before recording, go to *Windows Settings > Display*. If your *Scale and layout* or *Multiple displays* settings are different than what's recommended when recording, a warning message shows each time you record an image-based mouse click, and you must confirm the message.
+
+=== Using Image-Based Mouse Clicks
+
+To capture image-based mouse clicks:
+
+. Start a normal recording process.
+. While recording, press *F3* to start recording image-based mouse clicks, or press the *Image-Based Mouse Clicks* icon on the Recording Toolbar.
+. Capture an image by selecting it on the screen, accept or retake the image, and then set a mouse click, which the Recorder marks with a red dot. Record additional mouse clicks, such as double-mouse clicks, if desired.
+. Click *Finish*. This generates all relevant data for RPA Builder to create the _Search Pattern_ and _Mouse Action_ Action Steps for the recording.
+
+The process documentation and PDF generated by the Recorder include the recording data. Once you finish using image-based mouse clicks, you can continue with the normal recording process.
+
+When recording image-based mouse clicks in a Web Task:
+
+* Scrolling or zooming in the displayed web page or moving or resizing the browser window results in inaccurate relative clicks in the recording.
+* Any subsequent click after navigating to another web page can cause inaccuracies in the recording.
```

Comment:
suggestion: replace "Any subsequent click after navigating to another web page can cause inaccuracies in the recording." with "Navigating to a different web page during capturing the Image-Based Mouse Clicks (e.g. by clicking on a link) can cause inaccuracies in the recording of the subsequent relative mouse clicks."